src/utils.ts

```diff
@@ -0,0 +1,91 @@
+import { PresetInfo, PresetResult, NavigatorUABrandVersion } from "./types";
+
+export function some<T>(arr: T[], callback: (value: T, index: number) => any): boolean {
```

Comment:
Did you make a util to not use ployfill?

Reply:
@mixed 

yes. To support ie7.